src/screens/Gurbani/Navbar.tsx

```diff
@@ -31,10 +34,10 @@ const GurbaniNavbar = () => (
     left={<Icon style={styles.headerIcon} name="menu" />}
     right={<Icon style={styles.headerIcon} name="ios-options-outline" />}
     main={(
-      <>
-        <Image style={styles.logo} source={logo} />
-        <Typography variant="header" style={styles.heading}>Shabad OS</Typography>
-      </>
+      <View style={styles.logo}>
+        <Image style={styles.logoIcon} source={logo} />
+        <Typography variant="logo">Shabad OS</Typography>
```

Comment:
I don't think this should be a variant if the only time we use it is with "Shabad OS". Could be styled inline this component instead